build-logic/src/main/groovy/org/apache/groovy/gradle/ReleaseInfoGenerator.groovy

```diff
@@ -69,7 +69,7 @@ class ReleaseInfoGenerator extends DefaultTask {
 #  "License"); you may not use this file except in compliance
 #  with the License.  You may obtain a copy of the License at
 #
-#    http://www.apache.org/licenses/LICENSE-2.0
+#    https://www.apache.org/licenses/LICENSE-2.0
```

Comment:
In most places we have left "http" inside the license text. That was conservatively because that is what is actually in the license file we refer to and we didn't want to open up an even insignificantly small debatable legal point, but also waiting for tooling to catch up. I believe lots of other projects now use https in their headers and the tooling has for the most part caught up.

Reply:
So if I should revert this line?

Reply:
Reverted.

Reply:
Yeah, we'll no doubt probably eventually change this but we'll probably do all 5k places in one commit.